libr/bin/mangling/swift-sd.c

```diff
@@ -171,33 +177,44 @@ static char *swift_demangle_lib(const char *s) {
 	return NULL;
 }
 
+static const char *str_seek (const char *s, int n) {
```

Comment:
@trufae this seems useful to have in `r_str.h`, wdyt?

Reply:
well. the name looks confusing, doesnt returns null when seeking outside and dosnt handle negative values. not sure how much useful that function will be or in how many places we will be able to use this api. if its meaningful for other places, then yes, it can be added into r_str. i dont wanna expose methods that we dont need mainly

Reply:
@trufae This function would be useful in the Swift and MSVC demanglers, probably others too. I proposed another variant here https://github.com/radareorg/radare2/pull/20631 called `r_str_skip`. Since skipping negative bytes makes no sense.
IMHO String subslicing with clamped bounds is a common operation in other languages so I felt it's appropriate to add. But happy to leave it as static, lmk

Reply:
not for now